atomicapp/utils.py

```diff
@@ -377,6 +378,50 @@ def rm_dir(directory):
         distutils.dir_util.remove_tree(directory)
 
     @staticmethod
+    def getUid(user):
+        """
+        Get the UID of the specific user by grepping /etc/passwd unless
+        we are in a container.
+
+        Returns:
+            (int): User UID
+        """
+
+        # If we're in a container we should be looking in the /host/ directory
+        if Utils.inContainer():
+            os.chroot(HOST_DIR)
+            uid = pwd.getpwnam(user).pw_uid
+            os.chroot("../..")
+        else:
+            uid = pwd.getpwnam(user).pw_uid
+
+        return int(uid)
+
+    @staticmethod
+    def setFileOwnerGroup(src):
+        """
+        This function sets the correct uid and gid bits to a source
+        file or directory given the current user that is running Atomic
+        App.
+        """
+        user = Utils.getUserName()
+
+        # Get the UID of the User
+        uid = Utils.getUid(user)
+        gid = uid
```

Comment:
this is not a safe assumption to make. I think it would be safer if we actually did `pwd.getpwnam(user).pw_gid`. If `getUid()` is a new function then we could just change it to `getUidGid()` and have it return a tuple `(uid,gid)`.